inst/rmarkdown/templates/tufte_handout/resources/tufte-common.def

```diff
@@ -1,8 +1,8 @@
 %%
```

Comment:
It seems like we should also move the tufte support files from resources to skeleton for tufte handout, do you agree?